bh_core.sublime-settings

```diff
@@ -446,6 +446,17 @@
             "language_list": ["LaTeX", "LaTeX (TikZ)", "knitr (Rnw)"],
             "sub_bracket_search": "true",
             "enabled": true
+        },
+        // Pascal
+        {
+            "name": "pascal",
+            "open": "(?<=[\\s;])(try|class|record|begin|unit|repeat)\\b",
```

Comment:
A couple of things:

1. This will not catch try if it is on the first line.  Maybe something like this would work better? `(?:(?<=^)|(?<=[\\s;]))(try|class|record|begin|unit|repeat)\b`?
2. You should probably create a bh_plugin to handle pairing end and until to the proper opening brackets: see the luakeywords bh_plugin as an example: https://github.com/facelessuser/BracketHighlighter/blob/master/bh_modules/luakeywords.py.  I imagine `unitl` can only be linked to specific opening keyworkds as well as `end`.